### PR TITLE
Previous / Next links added to each page

### LINKED ISSUE #40: Page navigation DCNC - Next/Prior
In the old site we used to have a "next/prior" button set to navigate through the pages. With the loss of the guide always being on the side, I think the need is back. It could be anywhere on the page - such as in the header or footer for each page.

### PR DESCRIPTION
**Pull request by SteveLLamb**
Added pagination previous/next to the top and bottom of each page, closes #40

### CODE REVIEW

**Comment by SteveLLamb** [comment]
@japierce1028 added to https://isdcf.com/staging/general.html as an example

File: src/site/mobile.css [hunk 1]
```diff
@@ -31,6 +31,15 @@ pre {
   padding: 10px;
 }
 
+.page-link {
```
Comment (palemieux) [inline src/site/mobile.css:34]: Link is not grayed out on first and last page:

image

**Review by palemieux — CHANGES_REQUESTED** [review]
image

Navigation fails on local build. Suggest adding `--gh` parameter to `build.js` to trigger GH specific URLs.

File: src/site/mobile.css (hunk 1 above)
Comment (SteveLLamb) [reply src/site/mobile.css:34]: Removed buttons altogether for first and last pages in 07e34dc

**Review by palemieux — APPROVED** [review]

**Pull request merged**